.github/workflows/cleanup-firebase.yml

```diff
@@ -0,0 +1,29 @@
+name: Cleanup Firebase Metadata
+
+on:
+    schedule:
+        - cron: "24 18 * * 1" # Runs at 18:24 UTC every Monday
```

Comment:
out of curiosity, is there a reason you picked this time? it feels very specific

Reply:
I simply copied the line from another workflow `ci.yml`, this comment may answer the question:
```
 # (Since these CRONs are used by a lot of people -
 # let's be nice to the servers and schedule it _not_ on the hour)
```

Reply:
ahh gotcha!

Follow up question: I'm still not super familiar with firebase, will the performance of reading to / writing from the database be interrupted while this job is running? If so, it could be nice to have the job run in the middle of the night or sometime when we wouldn't be using it (18:24 UTC is like 11:24am pacific time)

Reply:
Since this job is to check data under the `results` collection, it shouldn't be interrupting any firebase's performances. But yeah we can totally schedule a time during non-working hours

Reply:
Seems like it shouldn't be an issue then, thanks for clarifying!